src/api-docs/source/index.html.md

```diff
@@ -45,18 +45,127 @@ search: true
 
 Welcome to the hsd API!
 
-The default hsd HTTP server listens on port (`12037` for main, `13037` for testnet, `14037` for regtest, and `15037` for simnet). It exposes a REST JSON, as well as a JSON-RPC API.
+hsd is a fork of [bcoin](https://github.com/bcoin-org/bcoin) and so if you are familiar with it,
+you'll find that the API is very similar to the [bcoin API](http://bcoin.io/api-docs/).
+bcoin was released with a JSON-RPC API that was based on the Bitcoin Core RPC interface.
+The actual Bitcoin Core RPC API has changed a lot since then, but if you are familiar with
+using bitcoind, you'll find a lot of the same RPC calls available in hsd.
+
+**There are four API servers in hsd**
+
+The full node and the wallet are separate processes in hsd, and their API servers run on separate ports.
+
+Network   | Wallet API Port | Node API Port
+--------- | --------------- | -------------
+main      | 12039           | 12037
+testnet   | 13039           | 13037
+regtest   | 14039           | 14037
+simnet    | 15039           | 15037
+
+In addition, each server has two API formats: JSON-RPC and a RESTful HTTP API.
+In general, the HTTP APIs respond to `GET` requests and the RPC APIs respond to
```

Comment:
Nit: I don't think we need this sentence `In general...`
There are a bunch of wallet HTTP endpoints that use `POST` and the RPC API has examples below.

Reply:
Something that might be more useful to state:

Much of the functionality is duplicated between the HTTP API and RPC API. Note that if multiple user-agents are using the wallet at the same time, the wallet RPC API is not safe as it maintains global state. It is recommended to use the wallet HTTP API in multiprocess/multithreaded environments.